acceptance/tests/options/config_file/command_line_override.rb

```diff
@@ -0,0 +1,39 @@
+test_name "flags set on the command line override config file settings"
+#
+# This test is intended to demonstrate that all command-line flags take precedence
+# over their counterparts in the config file. This only applies to exact repeats;
+# if a command-line flag conflicts with a config file setting, a warning will be
+# issued the same as if the user had specified two conflicting flags on the command
+# line.
+#
+config = <<EOM
+global : {
+    external-dir : "config/file/dir"
+}
+cli : {
+    debug : false
+}
+EOM
+
+agents.each do |agent|
+  step "Agent #{agent}: create config file"
+  config_dir = agent.tmpdir("config_dir")
+  config_file = File.join(config_dir, "facter.conf")
```

Comment:
this might not work on windows agents.  but we might get lucky because of cygwin (don't think i've ever said that before)